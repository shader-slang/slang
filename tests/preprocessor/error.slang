//TEST:SIMPLE:

// #error support

#define FIRST 0

#if FIRST
#error Not this one!
#else

#error This isn't valid!

#endif
