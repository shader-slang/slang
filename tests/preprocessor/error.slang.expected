result code = -1
standard error = {
tests/preprocessor/error.slang(11): error 15900: #error:  This isn't valid!
#error This isn't valid!
 ^~~~~
}
standard output = {
}
